test/powershell/engine/COM/COM.Basic.Tests.ps1

```diff
@@ -0,0 +1,35 @@
+
+try {
+    $defaultParamValues = $PSdefaultParameterValues.Clone()
+    $PSDefaultParameterValues["it:skip"] = !$IsWindows
+
+    Describe 'Basic COM Tests' -Tags "CI" {
+        It "Should enumerate ShellWindows" {
+            $shell = New-Object -ComObject "Shell.Application"
+            $windows = $shell.Windows()
```

Comment:
Good catch. NanoServer and IoT don't have the shell, so these tests don't apply. Will update.